translations/ui-orders/en.json

```
@@ -773,5 +773,28 @@
   "requests.request.isOpened": "Yes",
 
   "poLine": "PO line",
-  "poLine.pluralized": "{count} {count, plural, one {PO line} other {PO lines}}"
+  "poLine.pluralized": "{count} {count, plural, one {PO line} other {PO lines}}",
+  
+  "createdDateAndTime": "Created date and time",
+  "createdSource": "Created source",
+  "fund.fundIdentifier": "Fund ID",
+  "fund.selectAdjustment": "Select adjustment",
+  "notes.date": "Date",
+  "notes.type": "Type",
+  "poAdministrativeData": "PO admin data",
+  "poPurchaseOrder": "PO purchase order",
+  "polAdministrativeData": "POL admin data",
```

Rename "pol-" to "poLine-" and put it to the proper area, please
```suggestion
  "poLine.AdministrativeData": "POL admin data",
```